.github/workflows/integration-tests-dev.yml

```diff
@@ -5,6 +5,9 @@ env:
   TURBO_TEAM: ${{ vars.TURBO_TEAM }}
   POSTGRES_URL: ${{ secrets.POSTGRES_URL }}
   POSTGRES_URL_NON_POOLING: ${{ secrets.POSTGRES_URL_NON_POOLING }}
+  BLOB_READ_WRITE_TOKEN: ${{ secrets.BLOB_READ_WRITE_TOKEN }}
```

Comment:
For which store and team/project this token is? For Vercel Blob API I've used a token of one of my store but I guess we should use the same token and prefix our pathnames when necessary. Here's how it's done in Vercel Blob API: https://github.com/vercel/edge-functions/blob/26411a3960598d22ef7dbd79d3a92e57b0c2ca6e/packages/blob-storage-worker/test/index.test.ts#L17-L19

Reply:
> It's documented in the readme

Can you link this to me? Can't find it.

Reply:
https://github.com/vercel/storage/blob/main/test/next/README.md

Reply:
Can we be explicit and add a comment that says:

```
# This token can be found in Curated Tests > vercel-storage-next-integration-test-suite > storage > blob (which blob store is it?)
BLOB_READ_WRITE_TOKEN: ${{ secrets.BLOB_READ_WRITE_TOKEN }}
```

Reply:
It's extremely common for these E2E tests tokens to be opaque after 1 year and nobody knows where do they come from.